namer/namer.cc

```diff
@@ -703,7 +703,9 @@ class NameInserter {
         }
 
         auto members = onSymbol.data(ctx)->typeMembers();
-        auto it = absl::c_find_if(members, [&](auto mem) { return mem.data(ctx)->name == typeName->cnst; });
+        auto it = absl::c_find_if(members, [&](auto mem) {
```

Comment:
I don't understand what this does. Could you explain?
Why do you only compare `originalName`?